### PR TITLE
Corrected OAuth logout URL default

### PR DESCRIPTION
**Pull request by clintonb**
PLAT-1573

### CODE REVIEW

File: playbooks/roles/common_vars/defaults/main.yml [hunk 1]
```diff
@@ -211,7 +211,7 @@ COMMON_EXTRA_CONFIGURATION_SOURCES: []
 
 COMMON_OAUTH_PUBLIC_URL_ROOT: 'http://127.0.0.1:8000/oauth2'
 COMMON_OAUTH_URL_ROOT: '{{ COMMON_OAUTH_PUBLIC_URL_ROOT }}'
-COMMON_OAUTH_LOGOUT_URL: '{{ COMMON_OAUTH_PUBLIC_URL_ROOT }}/logout'
+COMMON_OAUTH_LOGOUT_URL: 'http://127.0.0.1:8000/logout'
```
Comment (jibsheet) [inline playbooks/roles/common_vars/defaults/main.yml]: Shouldn't http://127.0.0.1:8000 be in a var that then gets interpolated into PUBLIC_URL_ROOT and LOGOUT_URL?  Assuming we could then have only 1 to set when we turn it into courses.edx.org.

Comment (clintonb) [reply playbooks/roles/common_vars/defaults/main.yml]: Sure. Suggestions for the new variable name?

**Comment by clintonb** [comment]
Related: https://github.com/edx/edge-internal/pull/28

File: playbooks/roles/common_vars/defaults/main.yml (hunk 1 above)
Comment (feanil) [reply playbooks/roles/common_vars/defaults/main.yml]: `COMMON_OAUTH_BASE_URL`?

**Comment by clintonb** [comment]
More changes to edx-internal and edge-internal will be made once this is incorporated. This is fine to merge for now.

**Review by jibsheet — APPROVED** [review]

**Pull request merged**